hugr-py/src/hugr/tys.py

```diff
@@ -88,10 +88,8 @@ def to_serial(self) -> stys.BoundedNatParam:
 class OpaqueParam(TypeParam):
     """Opaque type parameter."""
```

Comment:
Update the doc at least, even if you are deciding not to rename the class. (Do we really avoid breakage by leaving it as `OpaqueParam`?? It seems odd not to rename when you *are* renaming the corresponding `OpaqueArg` -> `StringArg`)